src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php

```diff
@@ -323,11 +405,12 @@ private function fixAssertTrueFalse(Tokens $tokens, ArgumentsAnalyzer $arguments
 
             $isPositive = $isPositive ? 'positive' : 'negative';
 
-            if (false === self::$fixMap[$content][$isPositive]) {
+            $replace = self::$fixMap[$content][$isPositive];
+            if (false === $replace) {
                 return;
             }
 
-            $tokens[$assertCall['index']] = new Token([T_STRING, self::$fixMap[$content][$isPositive]]);
+            $tokens[$assertCall['index']] = new Token([T_STRING, (string) $replace]);
```

Comment:
`(string)` not needed after https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7997/files#r1641231813